capabilities/writetarget/beholder/report/datafeeds/id.go

```diff
@@ -0,0 +1,92 @@
+package datafeeds
```

Comment:
All of this report decoding:

- datafeeds
- mercury
- platform (e.g., CRE/Keystone)

should also eventually be extracted from here, moved to other more fitting locations, and then imported and injected back here to be used in WT processing.

Reply:
Created a ticket tracking this https://smartcontract-it.atlassian.net/browse/BCFR-1221